pkg/ddevapp/wordpress.go

```diff
@@ -194,6 +296,10 @@ func isWordpressApp(app *DdevApp) bool {
 	if _, err := os.Stat(filepath.Join(app.AppRoot, app.Docroot, "wp-login.php")); err == nil {
 		return true
 	}
+	// TODO: Add wildcard or ENV var to make more flexible, ie wordpress/
```

Comment:
I'll add better notes as to why we are doing the additional check.

We are looking for WP in a sub-directory, the earlier check was looking for it only in the docroot. Ideally it's a config settings we can control. All I'm adding is a check for WordPress in `wp`, but it could be in `wordpress` or whatever they want to call it by defining `WP_SITEURL`.

`define('WP_SITEURL', 'https://www.sitename.com/wp);`